apps/qubit/modules/repository/templates/browseSuccess.php

```diff
@@ -104,10 +104,8 @@
       </div>
     </div>
 
-    <div class="row">
-      <div class="span5">
-        <button id="toggleAdvancedFilters" class="btn fa fa-angle-double-down">&nbsp;<?php echo __('Advanced') ?></button>
-      </div>
+    <div id="content">
+      <a href="#" id="toggleAdvancedFilters" class="advanced-search-toggle"><?php echo __('Advanced search options') ?></a>
```

Comment:
Should "advanced search options" be wrapped for i18n? do we need aria attributes here for accessibility?